code/modules/economy/utils.dm

```diff
@@ -13,32 +13,20 @@
 		return acct
 
 
-/obj/proc/get_card_account(obj/item/card/I, mob/user=null, terminal_name="", transaction_purpose="", require_pin=0)
```

Comment:
`get_card_account` called in `/obj/machinery/vending/ui_data` and raises the error due to the (unreachable in practice) `input()`
None of the three callsites pass any arguments beyond the card